context/server.py

```diff
@@ -0,0 +1,43 @@
+import grpc
+from proto import user_pb2, user_pb2_grpc
+import db_client
+import client_interceptors
+import server_interceptors
+
+HOST = '[::]:50051'
+DB_TARGET = 'db.grpc.io:7777'
+
+class UserService(user_pb2_grpc.UserServiceServicer):
+    def __init__(self):
+        self._db_channel = grpc.intercept_channel(
+                grpc.aio.insecure_channel(DB_TARGET),
+                tracing_interceptors.tracing_interceptor())
+
+    async def CreateUser(self,
+                         request: user_pb2.CreateUserRequest,
+                         context: grpc.aio.ServicerContext):
```

Comment:
It would be nice to have also an example of how the timeout could be modified by the server evaluating the timeout provided by the context, for making a decision if the timeout needs to be modified or if it does not exist adding it.